src/main/java/org/fungover/storm/client/ClientHandler.java

```diff
@@ -22,11 +25,17 @@ public ClientHandler(Socket socket) {
     @Override
     public void run() {
         try {
-            out = new PrintWriter(clientSocket.getOutputStream(), true);
+            out = clientSocket.getOutputStream();
             in = new BufferedReader(new InputStreamReader(clientSocket.getInputStream()));
 
             String input = in.readLine();
-            out.println(input);
+
+            FileRequestHandler.handleRequest(input);
```

Comment:
This method handleRequest should return an object containing the header and body for multi threading to work.

Reply:
Add FileInfo object to handle multi threading